app/utils/jupyter/pipes.js

```diff
@@ -0,0 +1,40 @@
+import { Observable } from "rxjs";
+import { map, pluck, filter, take, mergeMap } from "rxjs/operators";
+import { executeRequest } from "@nteract/messaging";
+import { RECEIVE_EXECUTE_REPLY } from "../../epics/jupyterEpics";
+
+// Refactor this so command can be calculated either up stream or inside pipe
```

Comment:
Would be good to get Alex and Brian's ideas of what best practices are when it comes to custom pipes and where to put data